examples/fluids/qfunctions/newtonian_state.h

```diff
@@ -206,6 +207,18 @@ CEED_QFUNCTION_HELPER void FluxInviscid(NewtonianIdealGasContext gas, State s,
   }
 }
 
+CEED_QFUNCTION_HELPER void FluxInvisciddotNormal(NewtonianIdealGasContext gas, State s,
+                      const CeedScalar normal[3], StateConservative FluxdotNormal[3]) {
+  for (CeedInt i=0; i<3; i++) {
+    StateConservative Flux[3];
+    FluxInviscid(gas, s, Flux);
+    FluxdotNormal[i].density = s.U.momentum[i];
+    for (CeedInt j=0; j<3; j++)
+      FluxdotNormal[i].momentum[j] += Flux[i].momentum[j] * normal[i];
+    FluxdotNormal[i].E_total += (Flux[i].E_total) * normal[i];
+  }
+}
```

Comment:
The above is calling flux redundantly in each loop, doesn't initialize `FluxdotNormal` (which has the wrong shape), and is doing something that doesn't make sense for the density flux.

```suggestion
CEED_QFUNCTION_HELPER StateConservative FluxInviscidDotNormal(NewtonianIdealGasContext gas, State s,
                      const CeedScalar normal[3]) {
  StateConservative Flux[3], Flux_dot_n = {0};
  FluxInviscid(gas, s, Flux);
  for (CeedInt i=0; i<3; i++) {
    Flux_dot_n.density += Flux[i].density * normal[i];
    for (CeedInt j=0; j<3; j++)
      Flux_dot_n.momentum[j] += Flux[i].momentum[j] * normal[i];
    Flux_dot_n.E_total += Flux[i].E_total * normal[i];
  }
  return Flux_dot_n;
}
```